recipes/drf-generators/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
The license should be included in the next release but not the current one. We can remove this file on the feedstock once a new release has been made.